pom.xml

```diff
@@ -396,6 +399,11 @@
         <artifactId>svm</artifactId>
         <version>${graalapi.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Now that you declared smallrye here, I think it would make sense to also declare microprofile and micrometer here. I will fix myself.